src/main.py

```diff
@@ -464,4 +463,4 @@ def main():
 if __name__ == '__main__':
     logger.info('started working...')
     main()
-    logger.info('stopped working...')
+    logger.info('coinbot is offline', extra={"TELEGRAM": "STOPPED_WORKING"})
```

Comment:
@Alphavader I'd suggest changing the start bot message as well. In the same way, you've done it for stopping: add extra for telegram. And, please, make the message consistent: online - offline, start - stop

Reply:
will do!

Reply:
@iamtodor  should be done

Reply:
@Alphavader please, make the message consistent: coinbot is online - coinbot is offline. not like started working and coinbot is offline